Gemfile

```diff
@@ -188,7 +189,7 @@ gem 'lograge'
 gem 'rack-ssl-enforcer'
 
 # PubSub for NetSim
-gem 'pusher', '~> 0.14.5'
+gem 'pusher', '~> 1.0'
```

Comment:
Should I abandon https://github.com/code-dot-org/code-dot-org/pull/10281 and just leave this to you?